### DISCUSSION TITLE
export the 3D mask  (eg .nii or .stl)

### DISCUSSION THREAD

**Discussion by oscarramirezs** [Project-MONAI/MONAI#1667] [Q&A]
Hello,

can anybody help me?, I don't know if this subject has already been  in another post,

After training my model, can I export the 3D mask (eg .nii or .stl) that the model predicted?, is there a tutorial for this? 

Thanks for the help

Oscar

**Comment by Nic-Ma** [comment]
Hi @oscarramirezs ,

Thanks for your interest here.
You can take a look at this example: https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/torch/unet_evaluation_dict.py#L96

Thanks.
